gips/datahandler/worker.py

```diff
@@ -164,6 +164,14 @@ def _aggregate(job_id, outdir):
 
 def export_and_aggregate(job_id, export_kwargs):
     """Entirely TBD but does the same things as gips_project + zonal summary."""
+    with transaction.atomic():
+        job = dbinv.models.Job.objects.get(pk=job_id)
+        if job.status != 'pp-scheduled':
+            # TODO log/msg about giving up here
+            return job # not sure this is useful for anything
```

Comment:
This reminds me that we might want to give up more spectacularly on the workers, like `raise InvalidStateTransition` or something like that.